src/components/AmountForm.tsx

```diff
@@ -296,11 +305,11 @@ function AmountForm(
                     // eslint-disable-next-line react/jsx-props-no-spreading
                     {...rest}
                 />
-                {!!errorText && (
+                {!!rest.errorText && (
                     <FormHelpMessage
                         style={[styles.pAbsolute, styles.b0, canUseTouchScreen ? styles.mb0 : styles.mb3, styles.ph5, styles.w100]}
                         isError
-                        message={errorText}
+                        message={rest.errorText}
```

Comment:
this, on the other hand, was necessary, because the error handling was broken in general